dist/s_clang-scan.diff

```diff
@@ -14,14 +14,20 @@ src/conn/conn_capacity.c:291:5: warning: Value stored to 'capacity' is never rea
     capacity = steal_capacity = 0;
     ^          ~~~~~~~~~~~~~~~~~~
 1 warning generated.
-src/reconcile/rec_col.c:1050:23: warning: Null pointer passed as an argument to a 'nonnull' parameter
-                      memcmp(last.value->data, data, size) == 0)) {
-                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-1 warnings generated.
+src/reconcile/rec_col.c:1063:25: warning: Null pointer passed as an argument to a 'nonnull' parameter
+                        memcmp(last.value->data, data, size) == 0))) {
+                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1 warning generated.
```

Comment:
This is strange because your change did not modify `rec_col.c`. Why is this change in here?

Reply:
While updating the expected clang diff file, I generated the new diff and the line numbers are updated according to the code, so I update the line numbers.